modules/iam/groups/main.tf

```diff
@@ -0,0 +1,35 @@
+#tfsec:ignore:aws-iam-enforce-group-mfa
+resource "aws_iam_group" "admin" {
+  name = "Admin-group"
+  path = "/admins"
```

Comment:
let's verify the purpose of this [`path` attribute](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_group#path)
> [path](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_group#path) - (Optional, default "/") Path in which to create the group.

I think it will create the admin group under the `/admins` path, i.e. the end result will be `/admins/admin`. Is that what you expect? Otherwise, leaving the default path `/` is sufficent.

Reply:
Thanks for your sharing and explanation 🙏🏻 I just thought that we need to create the path/group on our own, so I decided to support many different paths. 

I removed it to use the default one in 8c28ff8 ✅